packages/v4/src/content/design-guidelines/components/helper-text/helper-text.md

```diff
@@ -0,0 +1,46 @@
+---
+id: Helper text 
+section: components
+---
+
+Helper text is an on-screen field guideline that helps provide context regarding field inputs. 
+<br/>
+## Usage:
+**Helper text** is most commonly used in conjuction with form elements to provide additional hints and information for the user. It can also be used to explain why certain information is being requested, how to use a specific input or to suggest helpful hints. Helper text should always be visible and appears as a subheading, unlike [popover](https://www.patternfly.org/v4/components/popover/design-guidelines/) which display content in a new window.
+<br/>
+### When to use helper text:
+* Use helper text to provide imperative information regarding an input, such as character limits, password expectations, etc.  
+* Use colored helper text to convey the status of a user's input. For example, if a user has met the character requirements, display helper text in green font to show success. Similarly, if a user has not met a password requirement, display the helper text in red to show failure/error. <br/>
+Example: 
+<img src="./img/errorhelpertext.png" alt="image showing error help text" width="452"/>
+<img src="./img/helpertextcolors.png" alt="image showing error help text" width="452"/>
+
+### When not to use helper text:
+* Do not use helper text to explain what a form field is or to provide additional form context. Instead, use a popover to convey any additional, non-imperative information to a user. Popover content is displayed in a new window after the user clicks or toggles over the popover icon. Its content consists of longer descriptions, formatted text, and complete sentences. 
+<br/>
```

Comment:
So I think you have to do </br> and not <br/> cause i'm not seeing a page break in the preview?

Reply:
Also bold here!